rmgpy/molecule/resonance.py

```diff
@@ -341,17 +389,211 @@ def generate_isomorphic_isomers(mol):
 
     return isomorphic_isomers
 
-def populate_resonance_generation_algorithm():
+def generateClarStructures(mol):
     """
-    A list with the current set of resonance generation algorithms.
+    Generate Clar structures for a given molecule.
+
+    Returns all Clar structures as a list.
```

Comment:
Its easier for other people if you give the object type for each input/output. For example you would say the output is a list of of :class:Molecule objects representing the Clar structures.